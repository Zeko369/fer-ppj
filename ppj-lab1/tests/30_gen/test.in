IDN 1 s
OP_PRIDRUZI 1 =
IDN 1 s
OP_PUTA 1 *
BROJ 1 1212
IDN 2 s
OP_PRIDRUZI 2 =
BROJ 2 1066
OP_PLUS 2 +
BROJ 2 307
OP_PUTA 2 *
IDN 2 s
IDN 3 a
OP_PRIDRUZI 3 =
BROJ 3 1176
OP_MINUS 3 -
BROJ 3 307
OP_PUTA 3 *
BROJ 3 50
IDN 4 e
OP_PRIDRUZI 4 =
BROJ 4 1006
OP_PUTA 4 *
IDN 4 a
IDN 5 z
OP_PRIDRUZI 5 =
BROJ 5 73
OP_PUTA 5 *
BROJ 5 505
