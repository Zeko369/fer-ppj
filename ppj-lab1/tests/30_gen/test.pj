s = s*1212
s = 1066 + 307*s
a = 1176 - 307*50
e = 1006*a
z = 73*505
