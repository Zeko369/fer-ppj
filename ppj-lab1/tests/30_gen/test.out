<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 s
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       IDN 1 s
      <T_lista>
       OP_PUTA 1 *
       <T>
        <P>
         BROJ 1 1212
        <T_lista>
         $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 s
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 1066
       <T_lista>
        $
      <E_lista>
       OP_PLUS 2 +
       <E>
        <T>
         <P>
          BROJ 2 307
         <T_lista>
          OP_PUTA 2 *
          <T>
           <P>
            IDN 2 s
           <T_lista>
            $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 a
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         BROJ 3 1176
        <T_lista>
         $
       <E_lista>
        OP_MINUS 3 -
        <E>
         <T>
          <P>
           BROJ 3 307
          <T_lista>
           OP_PUTA 3 *
           <T>
            <P>
             BROJ 3 50
            <T_lista>
             $
         <E_lista>
          $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 e
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          BROJ 4 1006
         <T_lista>
          OP_PUTA 4 *
          <T>
           <P>
            IDN 4 a
           <T_lista>
            $
        <E_lista>
         $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 5 z
        OP_PRIDRUZI 5 =
        <E>
         <T>
          <P>
           BROJ 5 73
          <T_lista>
           OP_PUTA 5 *
           <T>
            <P>
             BROJ 5 505
            <T_lista>
             $
         <E_lista>
          $
      <lista_naredbi>
       $
